### DISCUSSION TITLE
FindMemoryWord and the Night Watchman

### DISCUSSION THREAD

**Discussion by darrencubitt** [virtualagc/virtualagc#1214] [Q&A]
Hi Ron,

Just looking at how your code handles the Night Watchman (NEWJOB access), and you are comparing directly for a memory access at logical address 067. I know it probably wouldn't happen in real code, but would it not be possible to map erasable bank 0 into switched erasable and access NEWJOB that way? This code wouldn't catch that, but I'm not sure how the physical AGC handled this and am sure you would know better than me! In short: Is the NEWJOB trap based upon logical or physical address?

**Comment by rburkey2005** [comment]
@darrencubitt A bit late, I know, but unfortunately I didn't see this topic until just now, 3 years too late!  I expect that you're right that it's a loophole in yaAGC.  But it is hard to understand why anyone would do it that way.
